vaadinfrontend/src/main/java/life/qbic/datamanager/views/project/view/components/ProjectDetailsComponent.java

```diff
@@ -41,27 +47,51 @@ public class ProjectDetailsComponent extends Composite<CardLayout> {
 
   private final TextField titleField;
   private final FormLayout formLayout;
-  private final TextArea experimentalDesignField;
   private final TextArea projectObjective;
+  private final TextArea experimentalDesignField;
+
+  private final HorizontalLayout projectManagerLayout;
+  private final ContactElement pmElement;
```

Comment:
NIT: To keep the naming consistent
```suggestion
  private final ContactElement projectManagerElement;
```